readme.md

```diff
@@ -64,11 +65,11 @@ An example configuration file might look like below.  *Note:* The comments would
                     {
                         "old": {
                             "path": "c:/path/to/integration/connectionFile.sde", // Required if workspaces is defined
-                            "type": "SDE_WORKSPACE" // Optional, defaults to SDE_WORKSPACE
+                            "type": "SDE_WORKSPACE" // Optional, defaults to SDE_WORKSPACE, for file geodatabase, use "FILEGDB_WORKSPACE"
                         },
                         "new": {
                             "path": "c:/path/to/production/connectionFile.sde", // Required if workspaces is defined
-                            "type": "SDE_WORKSPACE" // Optional, defaults to SDE_WORKSPACE
+                            "type": "SDE_WORKSPACE" // Optional, defaults to SDE_WORKSPACE, for file geodatabase, use "FILEGDB_WORKSPACE"
```

Comment:
Might be worth a link here, since these aren't our code; these are just the parameters from the `replaceWorkspaces` method: http://desktop.arcgis.com/en/arcmap/10.3/analyze/arcpy-mapping/mapdocument-class.htm